homeassistant/components/matter/lock.py

```diff
@@ -89,21 +89,15 @@ async def send_device_command(
 
     async def async_lock(self, **kwargs: Any) -> None:
         """Lock the lock with pin if needed."""
-        code: str = kwargs.get(
-            ATTR_CODE,
-            self._lock_option_default_code,
-        )
+        code: str = kwargs.get(ATTR_CODE, None)
```

Comment:
"None" is redundant here, since None is the default return value of get.

Reply:
Thanks! Done.